Description: Ensure revision is never <0
 The revision in the code must be >0 as it is stored in a UINT16 variable.
 This change sets the subversion revision into the makefile.
Author: Seb James <seb.james@sheffield.ac.uk>
Forwarded: By email to Ben Michinson, the upstream developer.
Last-Update: 2014-02-20
--- a/makefile.brahms
+++ b/makefile.brahms
@@ -116,7 +116,7 @@
 VERSION_BRAHMS_REV=${shell svnversion -n . | sed s/.*:// | sed s/M//}
 endif
 ifeq ($(VERSION_BRAHMS_REV),exported)
-	VERSION_BRAHMS_REV=-1
+	VERSION_BRAHMS_REV=2695
 endif
 
 # SystemML version
